cfme/tests/cloud/test_keypairs.py

```diff
@@ -29,7 +30,6 @@ def keypair(appliance, provider):
 
 @pytest.mark.meta(blockers=[BZ(1718833, forced_streams=["5.10", "5.11"],
                                unblock=lambda provider: provider.one_of(OpenStackProvider))])
```

Comment:
This BZ is closed.
```
In [1]: BZ(1718833, forced_streams=["5.10", "5.11"]).blocks                                                                                                                  
.....
....
...
Found matching bug for 1718833 by release - #1718833
Out[1]: False
```
- Can we remove the **blockers** marker?

Reply:
Agree, we should remove this @nachandr, and include it as `automates` or docblock metadata.

Reply:
Done